packages/runtime/package.json

```diff
@@ -109,9 +114,9 @@
   "peerDependencies": {
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
-    "next": "^13.4.0 || ^14.0.0",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "next": "^13.4.0 || ^14.0.0 || >=15.0.0-rc",
```

Comment:
Should we support release candidates? Also, should we use `>=` here? We wouldn't want Next.js 16 to automatically be a supported peer dependency until we explicitly support it, right?

I would say `^15.0.0` for Next.js and `^19.0.0 || 19.0.0-rc` for React